bin/reth/src/args/rpc_server_args.rs

```diff
@@ -78,6 +89,76 @@ impl RpcServerArgs {
             }
         }
     }
+
+    pub(crate) async fn server<Client, Pool, Network>(
+        &self,
+        client: Client,
+        pool: Pool,
+        network: Network,
+    ) -> Result<RpcServerHandle, RpcError>
+    where
+        Client: BlockProvider + HeaderProvider + StateProviderFactory + Clone + 'static,
+        Pool: TransactionPool + Clone + 'static,
+        Network: NetworkInfo + Peers + Clone + 'static,
+    {
+        reth_rpc_builder::launch(
+            client,
+            pool,
+            network,
+            self.transport_rpc_module_config(),
+            self.rpc_server_config(),
+        )
+        .await
+    }
+
+    fn transport_rpc_module_config(&self) -> TransportRpcModuleConfig {
+        let mut config = TransportRpcModuleConfig::default();
+        let rpc_modules =
+            RpcModuleSelection::Selection(vec![RethRpcModule::Admin, RethRpcModule::Eth]);
```

Comment:
for a follow-up, but we should also add these to the cli config

Reply:
i can't get you.

Reply:
I mean that Admin & Eth should not always be enabled by default, the user should specify which apis they want to enable. 

@mattsse I also wonder if we should ship some solution for having private / public APIs, e.g. exposing the Admin API on a separate port that is not exposed to the internet